action.yml

```diff
@@ -5,7 +5,6 @@ runs:
   using: "node12"
   main: "dist/restore/index.js"
   post: "dist/save/index.js"
-  post-if: "success()"
```

Comment:
Oh wow, so `always()` is the default? Even better.

Reply:
Yep!